src/System.Device.Gpio/System/Device/Gpio/Processor.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace System.Device.Gpio.System.Device.Gpio
+{
+    /// <summary>
+    /// The board processor.
+    /// </summary>
+    public enum Processor
```

Comment:
Maybe I'm blind, but this seems to be unused.

Reply:
Yes, you're right. I used it into my driver, I would like to use it in the future to better identify the board

Reply:
There is also a confusion in BCM nomenclature. The 3 versions of the chip follow the following  scheme. There are now three pieces of silicon, BCM2708, BCM2709 and BCM2710 (processors) and three packages BCM2835, BCM2836 and BCM2837 (processors packaged with DRAM). Sometimes we read the formers sometimes the latters